### PR TITLE
3 feature a user can sign up

### PR DESCRIPTION
**Pull request by BenjaminNeustadt**
Had some issues with git, currently at the stage of creating a user that can sign up.

### CODE REVIEW

**Review by robimaliqi — APPROVED** [review]
approved

**Review by robimaliqi — APPROVED** [review]
approved

File: spec/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  
+      user = User.create('Godzilla', 'godzilla@email.com', '1234')
+
+      expect(user).to include('Godzilla')
+      expect(user).to include('godzilla@email.com')
+      expect(user).to include('1234')
+    end
+  end
+end
+
```
Comment (Hafizalimi7) [inline spec/user_spec.rb:27]: maybe, more test ?

File: spec/features/sign_up_spec.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      fill_in 'username', with: 'Godzilla'
+      fill_in 'email', with: 'godzilla@email.com'
+      fill_in 'password', with: '1234'
+      click_button 'Sign Up'
+
+      expect(page).to have_content "Welcome Godzilla!"
+      expect(current_path).to eq '/users/sign_up'
+    end
+  end
+end
```
Comment (Hafizalimi7) [inline spec/features/sign_up_spec.rb:25]: very clean, and understandable code

File: db/migrations/02_create_peeps_table.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+  id SERIAL PRIMARY KEY,
+  content VARCHAR(60) NOT NULL,
+  user_id int,
+  time_stamp timestamp NOT NULL DEFAULT NOW(),
+  CONSTRAINT fk_user_id FOREIGN KEY (user_id) REFERENCES users (id)
+);
+
+
+-- CREATE TABLE peeps(id SERIAL PRIMARY KEY, content VARCHAR(60), user_id VARCHAR(20), time_stamp timestamp);
```
Comment (Hafizalimi7) [inline db/migrations/02_create_peeps_table.sql:11]: very good attention to detail

**Pull request merged**